setup.py

```diff
@@ -217,7 +217,7 @@
 
 setup(
     name="datasets",
-    version="1.12.1.dev0",  # expected format is one of x.y.z.dev0, or x.y.z.rc1 or x.y.z (no to dashes, yes to dots)
+    version="1.12.1.dev1",  # expected format is one of x.y.z.dev0, or x.y.z.rc1 or x.y.z (no to dashes, yes to dots)
```

Comment:
why this change ?

Reply:
I was not sure if every commit must increase the version or not.

Reply:
Note that in another project I depend on a commit of the git repository of datasets:

https://github.com/huggingface/datasets-preview-backend/blob/10fd611042b41f2c8bed767a080f3a57b4429653/pyproject.toml#L28

And the dependency is updated only if the version increases. Without this, I have to do:

```
poetry remove datasets
```

add the line manually to pyproject.toml with the new commit, then

```
poetry lock
poetry install
```

Reply:
I will revert this.